src/main/java/at/hannibal2/skyhanni/features/mining/glacitemineshaft/MineshaftWaypointType.kt

```diff
@@ -0,0 +1,14 @@
+package at.hannibal2.skyhanni.features.mining.glacitemineshaft
+
+import at.hannibal2.skyhanni.utils.LorenzColor
+
+// could merge with CorpseAPI (besides the last 3 types)
```

Comment:
this is fine to stay here, comment can be removed

Reply:
meant CorpeType, they're very similar